actions/upload-metadata/action.yml

```diff
@@ -52,12 +54,27 @@ runs:
           "stacks": ${{ inputs.stacks }},
           "source": "${{ inputs.source-uri }}",
           "source_sha256": "${{ inputs.source-sha256 }}",
-          "deprecation_date": "${{ inputs.deprecation-date }}"
+          "deprecation_date": "${{ inputs.deprecation-date }}",
+          "created_at": "${timestamp}",
+          "modified_at": "${timestamp}"
         }
         EOF
         )"
 
-        updated_metadata="$(jq --argjson metadata "${metadata}" --arg version "${{ inputs.version }}" -r 'del(.[] | select(.version == $version)) | [$metadata] + .' metadata.json)"
+        original_created_at="$(jq \
+          --arg version "${{ inputs.version }}" \
+          -r '.[] | select(.version == $version) | .created_at // ""' metadata.json)"
+
+        if [[ -n "${original_created_at}" ]]; then
+          metadata="$(echo "${metadata}" | jq \
+            --arg created_at "${original_created_at}" \
+            -r '.created_at = $created_at')"
+        fi
```

Comment:
I think it might be a bit clearer what we're doing if we calculate the correct `created_at` timestamp before putting it in the metadata JSON, rather than putting one timestamp in and then potentially overwriting it in that if statement.

Happy to approve as is, just wondering what you thought of that.

Reply:
Agreed, pushed a new commit with that change